WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StatsTimeRangeV4+UI.swift

```diff
@@ -79,25 +79,4 @@ extension StatsTimeRangeV4 {
             return startDate.startOfDay(timezone: siteTimezone)
         }
     }
-
-    /// Returns a date formatter for the x-axis labels of a stats chart
-    ///
-    /// - Parameter siteTimezone: site time zone, which the stats data are based on
-    func chartDateFormatter(siteTimezone: TimeZone) -> DateFormatter {
-        let dateFormatter: DateFormatter
-        switch intervalGranularity {
-        case .hourly:
-            dateFormatter = DateFormatter.Charts.chartAxisHourFormatter
-        case .daily:
-            dateFormatter = DateFormatter.Charts.chartAxisDayFormatter
-        case .weekly:
-            dateFormatter = DateFormatter.Charts.chartAxisDayFormatter
-        case .monthly:
-            dateFormatter = DateFormatter.Charts.chartAxisMonthFormatter
```

Comment:
nit: `chartAxisMonthFormatter` and `chartAxisHourFormatter` are not used anywhere else. Should we consider deleting those from `WooCommerce/Classes/Extensions/DateFormatter+Helpers.swift`?

Reply:
Nice catch! Removed in 26a5371d46b308e2b1ee2a714be2dd2ebfe0479a.